### PR TITLE
Adds multimeter, the nerfed multitool

### PR DESCRIPTION
**Pull request by nicbn**
### Multitool wasn't nerfed

[Changelogs]: #
:cl:
add: Added the multimeter (currently found nowhere)
/:cl:

Adds the multimeter, a tool which use currently is only measure power on powernets. It already has a sprite, but can't be found anywhere. This can be used, for example, in ghost roles where engineers can't have a multitool as it is kinda overpowered or to replace multitools in all-access storage, so people can't run around pulsing doors.

**Calling multitool_act won't call multimeter_act by default**

### CODE REVIEW

**Comment by ShizCalev** [comment]
Atomize please :(

**Comment by arsserpentarium** [comment]
and don't nerf multitool.

**Comment by optimumtact** [comment]
how is a multitool overpowered?

**Comment by nicbn** [comment]
I didn't nerf multitool. All multimeter can do the multitool can do too.

Multitool isn't overpowered for normal gameplay, but you might want some ghost role to be unable to use some multitool abilities. More abilities can be added to the multimeter as they are needed. Also multimeter can be added in ruins as some kind of ancient multitool.

IN OTHER WORDS: This PR fixes two sprites. Gameplay-wise, this PR does nothing. It adds something that can be implemented later on.

**Comment by davethwave** [comment]
This would work with the hivebot station at least.

**Comment by nicbn** [comment]
Please notice that multitool_act won't call multimeter_act by default.

**Comment by Dorsisdwarf** [comment]
something something prs should be more atomic?

File: code/game/atoms.dm [hunk 1]
```diff
@@ -538,7 +538,9 @@
 	switch(tool_type)
 		if(TOOL_CROWBAR)
 			return crowbar_act(user, tool)
-		if(TOOL_MULTITOOL)
+		if(TOOL_MULTIMETER)
+			return multimeter_act(user, tool)
+    	if(TOOL_MULTITOOL)
```
Comment (ShizCalev) [inline code/game/atoms.dm]: TABS.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Review by ShizCalev — CHANGES_REQUESTED** [review]
Please split these up into separate PRs.

**Comment by optimumtact** [comment]
I cannot think of a single reason for a ghost role to not access multitools

**Comment by nicbn** [comment]
@optimumtact setting up telecomms, hacking

**Comment by ACCount12** [comment]
Even if this PR wasn't such a mess, this is just unnecessary.

**Comment by Spaceinaba** [comment]
who cares if they can set up telecomms they cant link it to the station anyways

**Comment by nicbn** [comment]
Closing this mess, if anyone wants to use the multimeter sprite, the license is CC0, go on

**Pull request closed without merging**